Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/Source/AtomViewportDisplayInfoSystemComponent.cpp

```diff
@@ -284,14 +284,19 @@ namespace AZ::Render
         }
 
         const double frameIntervalSeconds = m_fpsInterval.count();
+        constexpr float convertToInfAbove = 10000.0f;
+        const AZStd::string averageFpsString = averageFPS > convertToInfAbove ? "inf" : AZStd::string::format("%.1f", averageFPS);
+        const AZStd::string minFpsString = minFPS > convertToInfAbove ? "inf" : AZStd::string::format("%.0f", minFPS);
+        const AZStd::string maxFpsString = maxFPS > convertToInfAbove ? "inf" : AZStd::string::format("%.0f", maxFPS);
+        const AZStd::string averageFrameMsString = averageFrameMs > convertToInfAbove ? "inf" : AZStd::string::format("%.1f", averageFrameMs);
```

Comment:
suggestion/nit: This is very minor but we could wrap up the logic into a small inline function we can use if we ever want to update this (theoretically could be tested too 😉), something like...

```c++
AZStd::string ClampedFloatDisplay(float input, float upperLimit)
{
    return input > upperLimit ? "inf" : AZStd::string::format("%.0f", input);
}
```

Reply:
Changed to a lambda

Reply:
Nice! There's a chance a compiler might yell at you about the format string but hopefully it'll be okay, thanks for updating! 😄